deploy/upstream/quickstart/crds.yaml

```diff
@@ -1,10848 +0,0 @@
----
```

Comment:
looks like we lost this file - it's possible the release script hasn't been updated since the crds were moved?

Reply:
@ecordell could you give a little bit more context for this?